co-log/co-log.cabal

```diff
@@ -52,6 +52,7 @@ library
                      , stm >= 2.4 && < 2.6
                      , text ^>= 1.2.3
                      , time >= 1.8 && < 1.10
```

Comment:
`chronos` should replace `time` dependency, so I guess this library can be safely removed from `build-depends`